app/controllers/room_controller.rb

```diff
@@ -0,0 +1,9 @@
+class RoomController < ApplicationController
+
+  def create
+    
+   guest_room = GuestRoom.new(guest_id: params[:id], room_id: params[:room_id])
+   guest_room.save
+   redirect_to "/guests/#{params[:id]}"
+  end
```

Comment:
I would move this logic to a joins controller like, `GuestRoomsController` instead of having it here like this.